scripts/sql-test-helpers/docker-compose-postgresql.yaml

```diff
@@ -0,0 +1,12 @@
+services:
+  postgres:
+    image: postgres:14-alpine
```

Comment:
Would it be possible to leverage an image that has the [pgvector](https://github.com/pgvector/pgvector) extension and store the embeddings in a vector column. Ideally leverage a vector-specific query against it in the ingest destination test.